sentry/src/main/java/io/sentry/util/JsonSerializationUtils.java

```diff
@@ -0,0 +1,37 @@
+package io.sentry.util;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicIntegerArray;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+@ApiStatus.Internal
+public final class JsonSerializationUtils {
+
+  public static @NotNull Map<String, Object> calendarToMap(final @NotNull Calendar calendar) {
+    final @NotNull Map<String, Object> map = new HashMap<>();
+
+    map.put("year", (long) calendar.get(Calendar.YEAR));
```

Comment:
Just a question to understand
why are you casting to long if it is a `Map<String, Object>`? integers should be fine anyway, right?

Reply:
@adinauer everything else looks good. Just these 2 questions, then you can merge

Reply:
Updated